cypress/page/payment-step.page.ts

```diff
@@ -7,7 +7,7 @@ class PaymentPage {
     constructor() {
         this.payByBankWireButton = ".bankwire";
         this.confirmButton = "#cart_navigation > .button > span";
-        this.confirmeMessage = "#center_column > div > p > strong";
+        this.confirmeMessage = "#center_column > .box > .cheque-indent > .dark";
```

Comment:
what if instead of specifying parentship relations with `>`, you just specify the classes without defining any specific relationship, just that the should be withing the #center_column:
`#center_column .cheque-indent strong`

Reply:
Done!